test/Microsoft.ML.Core.Tests/Microsoft.ML.Core.Tests.csproj

```diff
@@ -2,6 +2,8 @@
   <PropertyGroup>
     <DefineConstants>CORECLR</DefineConstants>
     <StrongNameKeyId>Test</StrongNameKeyId>
+    <!-- Temporary workaround for https://github.com/dotnet/roslyn/issues/64365 -->
+    <LangVersion>10.0</LangVersion>
```

Comment:
This can be removed now that we're using a newer SDK.

Reply:
Appears to still be broken in RC2, I think we need a GA compiler to get the fix.